lib/screens/forgotpassword/forgotpassword.screen.dart

```diff
@@ -0,0 +1,166 @@
+  final _formkey = GlobalKey<FormState>();
+
+  resetPassword() async {
+    try {
+      await FirebaseAuth.instance.sendPasswordResetEmail(email: email);
+      ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+          content: Text(
+            "Password Reset Email has been sent !",
+            style: TextStyle(fontSize: 20.0),
+          )));
+    } on FirebaseAuthException catch (e) {
+      if (e.code == "user-not-found") {
+        ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+            content: Text(
+              "No user found for that email.",
+              style: TextStyle(fontSize: 20.0),
+            )));
+      }
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: Container(
+          decoration: const BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage("assets/SL-051919-20420-09.jpg"),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Container(
+            margin: const EdgeInsets.all(24),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                _header(context),
+                _inputField(context),
+                _signup(context),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  _header(BuildContext context) {
+    return Row(
+      children: [
+        Image.asset(
+          "assets/logo.png", // Replace with your image asset path
+          height: 90, // Adjust the height as needed
+        ),
+        const SizedBox(width: 10), // Add some space between the logo and text
+        Expanded(
+          child: LayoutBuilder(
+            builder: (context, constraints) {
+              final maxWidth = constraints.maxWidth;
+              final textStyle = TextStyle(
+                fontSize: 30.0,
+                fontWeight: FontWeight.bold,
+                color: Color(0xFF36A383),
+              );
+
+              final welcomeText = Text(
+                "Welcome to",
+                style: textStyle.copyWith(fontSize: maxWidth / 8),
+                overflow: TextOverflow.ellipsis,
+              );
+
+              final recycleCashText = Text(
+                "RecycleCash",
+                style: textStyle.copyWith(fontSize: maxWidth / 8),
+                overflow: TextOverflow.ellipsis,
+              );
+
+              return Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [welcomeText, recycleCashText],
+              );
+            },
+          ),
+        ),
+      ],
+    );
+  }
+
+  _inputField(BuildContext context) {
+    return Form(
+      key: _formkey,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          TextFormField(
+            validator:(value){
+              if(value==null||value.isEmpty){
```

Comment:
aici ar mai merge sa verificati si formatul mailului ;)
vezi aici --> [https://stackoverflow.com/questions/16800540/how-should-i-check-if-the-input-is-an-email-address-in-flutter](url)